interpreter/cling/lib/Interpreter/IncrementalCUDADeviceCompiler.cpp

```diff
@@ -7,8 +7,7 @@
 // LICENSE.TXT for details.
 //------------------------------------------------------------------------------
 
-#include "IncrementalCUDADeviceCompiler.h"
-
+#include "cling/Interpreter/IncrementalCUDADeviceCompiler.h"
```

Comment:
Why is this now public? The commit log
```
- make public that it is accessible via gCling object during the Cling
runtime.
```
says that it's made accessible to make it accessible ;-) Can you give a reason? Testing / ...

Reply:
I made it public to get information from device interpreter instance during runtime. For example, which arguments are set, like include paths, optimization level etc. I wanted the same like `gCling->getOptions().CompilerOpts.Remaining;` for the device compiler. 

In future, I also plan to give access to the ptx code (GPU assembler code) for each transaction.